zeek-ocsf/package.yaml

```diff
@@ -485,22 +485,26 @@ pipelines:
         protocol_name: "udp",
         protocol_num: 17,
       }
-      if zeek.client_addr.is_v6() or zeek.client_addr.is_v6() {
+      if not zeek.id?.resp_h? == null {
```

Comment:
Why does `id?` have a trailing question mark?

Reply:
The question mark makes sense iff `zeek.id` is not present in all events. Whether that is the case or not, I don't know. The following `else if` doesn't need it because if `zeek.id.resp_h` does not exist, then we will always go into the first case anyway.

Reply:
That is right, but I believe the `not` is misplaced.

Reply:
https://github.com/tenzir/library/pull/87